### ISSUE TITLE
Callback on timeout

### ISSUE THREAD

**Issue by campaignium** [digitalfortress-tech/localstorage-slim#10]
Could you add a feature to allow for a callback function to be passed into the set() method, and this callback would be fired when the ttl is reached?

Thanks
Justin

**Comment by niketpathak** [comment]
yes that is possible, we could allow callbacks to be optional. we will try to include this in our next **minor** release.

**Comment by Thomas-1985** [comment]
+1
